tests/test_simple.py

```diff
@@ -216,168 +186,137 @@ def test_density(self):
         dens_s = pa.density.bins("output/dens_cyl_s.h5", target_dens=16)
         dens_p = pa.density.bins("output/dens_cyl_p.h5", target_dens=16)
         dens_s_mean = pa.density.mean(dens_s)
+        pa.density.bins_plot(dens_s, target_dens=0.146, is_mean=True)
+        plt.savefig("output/density.pdf", format="pdf", dpi=1000)
 
         dens_no_remove = pa.density.bins("output/dens_cyl_no_remove.obj")
-
         dens_slit = pa.density.bins("output/dens_slit.h5", target_dens=997)
         dens_box = pa.density.bins("output/dens_box.h5")
-
         # Plot density
         plt.figure()
         pa.density.bins_plot(dens_s, target_dens=0.146)
         pa.density.bins_plot(dens_s, target_dens=0.146, is_mean=True)
         plt.savefig("output/density.pdf", format="pdf", dpi=1000)
         # plt.show()
-
         plt.figure()
         pa.density.bins_plot(dens_no_remove)
         plt.savefig("output/density_no_remove.pdf", format="pdf", dpi=1000)
         # plt.show()
-
         plt.figure()
         pa.density.bins_plot(dens_slit, is_mean=True)
         plt.savefig("output/density_slit.pdf", format="pdf", dpi=1000)
         # plt.show()
-
         plt.figure()
         pa.density.bins_plot(dens_s, intent="in")
         pa.density.bins_plot(dens_p, intent="in")
         plt.legend(["Single core", "Parallel"])
         plt.savefig("output/density_in.pdf", format="pdf", dpi=1000)
-        # plt.show()
-
+        plt.show()
```

Comment:
plt.show immer auskommentieren in den tests, sonst kann es bei der automatisierung zu prblemen führen